src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -14,8 +14,10 @@
     <ToolHost Condition="'$(OS)' != 'Windows_NT'">corerun</ToolHost>
 
     <ToolTargetFramework Condition="'$(ToolTargetFramework)' == ''">DNXCore,Version=v5.0</ToolTargetFramework>
+    <ToolOS Condition="'$(OS)' == 'Windows_NT'">win7</ToolOS>
+    <ToolOS Condition="'$(OS)' != 'Windows_NT'">ubuntu.14.04</ToolOS>
```

Comment:
We probably should go ahead and add osx. We should figure out how we make this extensible so that people can do this without requiring a build tools update, at least temporarily. I wonder if we should generate this project.json file on the fly and include the runtime id's based on what is setup and then generate the lock file at that point so we don't have to keep updating the lock file for all possible runtimes.